src/test/java/uk/gov/hmcts/ccd/domain/service/common/DefaultEndpointAuthorisationServiceTest.java

```diff
@@ -49,13 +49,13 @@ void shouldReturnTrueWhenLoggedInUserHasCAARole() {
     }
 
     @Test
-    void shouldReturnTrueWhenUserHasSolicitorRoleAndUserHasAccessToCase() {
+    void shouldReturnFalseWhenUserHasSolicitorRoleAndUserHasAccessToCase() {
         CaseDetails caseDetails = mock(CaseDetails.class);
         when(this.userRepository.getUserRoles()).thenReturn(Sets.newHashSet("caseworker-probate-solicitor"));
         when(this.caseAccessService.userCanOnlyAccessExplicitlyGrantedCases()).thenReturn(true);
```

Comment:
We don't need to mock the method userCanOnlyAccessExplicitlyGrantedCases in any of the tests here as it is removed.